edk2toolext/invocables/edk2_ci_build.py

```diff
@@ -113,14 +113,21 @@ def GetLoggingLevel(self, loggerType):
         return logging.DEBUG
 
     def AddCommandLineOptions(self, parser):
-
-        parser.add_argument('-p', '--pkg', '--pkg-dir', dest='packageList', nargs="+", type=str,
-                            help='A package or folder you want to test (abs path or cwd relative).  '
-                            'Can list multiple by doing -p <pkg1> <pkg2> <pkg3>', default=[])
+        '''  Add command line options to the argparser '''
+        # This will parse the packages that we are going to build
+        parser.add_argument('-p', '--pkg', '--pkg-dir', dest='packageList', type=str,
+                            help='A package or folder you want to test (abs path or cwd relative).'
+                            'Can list multiple by doing -p <pkg1>,<pkg2> or -p <pkg3> -p <pkg4>. It must end in Pkg.',
+                            action="append")
 
     def RetrieveCommandLineOptions(self, args):
         '''  Retrieve command line options from the argparser '''
-        self.packageList = args.packageList
+        packageListSet = set()
+        for item in args.packageList:  # Parse out the individual packages
+            item_list = item.split(",")
```

Comment:
I would suggest using strip() as well to make sure there is no whitespace at beginning or end.